scripts/profile_epoch.py

```diff
@@ -0,0 +1,81 @@
+import torch
+from modelforge.train.training import return_toml_config
+from typing import Dict, Any
+
+
+def perform_training(trainer, model, dm):
+
+    # Run training loop and validate
+    trainer.fit(
+        model,
+        train_dataloaders=dm.train_dataloader(),
+        val_dataloaders=dm.val_dataloader(),
+    )
+
+
+def setup(model_name: str):
+    from modelforge.dataset.utils import RandomRecordSplittingStrategy
+    from lightning import Trainer
+    from modelforge.potential import NeuralNetworkPotentialFactory
+    from modelforge.dataset.dataset import DataModule
+
+    config = return_toml_config(
+        f"../modelforge/tests/data/training_defaults/{model_name.lower()}_qm9.toml"
```

Comment:
I think we should change this to use module specified paths, not relative. 

So something like

```
from importlib import resources
from modelforge import tests as modelforge_tests


config = return_toml_config(
        f"{resources.files(modelforge_tests)}/data/training_defaults/{model_name.lower()}_qm9.toml"
)
```